docs/get-started/get-started.md

```diff
@@ -45,7 +45,7 @@ The current mapbox-gl release requires its stylesheet be included at all times.
 You may add the stylesheet to the head of your page:
 ```html
 <!-- index.html -->
-<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.0/mapbox-gl.css' rel='stylesheet' />
+<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css' rel='stylesheet' />
```

Comment:
Following the discussion on the issue, maybe something along the lines of:

`https://api.tiles.mapbox.com/mapbox-gl-js/v<YOUR_MAPBOX_VERSION>/mapbox-gl.css`

Find out your mapbox version by running `yarn list mapbox-gl` or `npm ls mapbox-gl`.

Reply:
👍 I'll make the change to the code snippet.  That command doesn't work for me though for styles:
```
❯ yarn list mapbox-gl
yarn list v1.16.0
warning Filtering by arguments is deprecated. Please use the pattern option instead.
└─ mapbox-gl@1.2.0
```

Is there an easy way to see what version of the css you should use based on the package version?

Reply:
An alternative is to take that whole block out.  react-map-gl automatically adds the css if you don't add the link tag.  I didn't know that when I first started.

Reply:
> That command doesn't work for me though for styles

You should use the same version of css as the JavaScript dependency. In this case 1.2.0. Is it still giving you the warning?

Reply:
Yep, that works!  I was dropping the `v` from the URL and thought the versions didn't line up 🤦‍♂️

I pushed a new commit that makes these changes.